src/control/security/auth/auth_sys.go

```diff
@@ -66,228 +64,6 @@ func VerifyToken(key crypto.PublicKey, token *Token, sig []byte) error {
 	return errors.Wrap(err, "token verification Failed")
 }
 
-func sysNameToPrincipalName(name string) string {
```

Comment:
This file was intended to encapsulate the AUTH_SYS authentication token type (i.e. using unix usernames/groups). Rather than breaking this up, I think you should consider the flow for this new authentication type separately. If you're free to share any details on the protocol for the access manager, I'd be happy to collaborate.